### DISCUSSION TITLE
I can't push changes?

### DISCUSSION THREAD

**Discussion by LordKallig** [community/community#65740] [New to GitHub] [answered]
I'm sure my password is correct, but it still won't let me.

**Comment by arafajs** [comment]
- Make sure you have write access to the repository you are trying to push to.
- Make sure the branch you are trying to push to is up-to-date with the remote branch.
- Make sure you are using the correct git command to push the changes.

**Comment by Arkenstine** [comment, marked as answer]
Make sure you have access to the repository.

Also, are you trying to push from the terminal? You need a personal access token not your password. 
https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens
